src/Containers/App/index.js

```diff
@@ -11,11 +11,11 @@ export default () => (
       <Suspense fallback={<div />}>
         <Switch>
           {
-            config.artists.map(({slug, component}) => {
+            config.artists.map(({slug, component}, i) => {
               const Comp = lazy(() => import(`../../work/${component}`));
               return (
                 <Route path={`/${slug}`} key={slug}>
-                  <Comp />
+                  <Comp config={config.artists[i]}/>
```

Comment:
Passing the config data to the artist component to automatically infer the path and title.

Reply:
i'm not sure if this will do what is intended. Lazy is used to code split the bundle, so each artist has their own js file instead of loading them all at once. Or is this just to pass the SEO metadata?

Reply:
This is to just pass the SEO data from the config to the component. I'm not sure if what I'm doing is right. But the styling issue existed even when I had the metadata hardcoded with this file unmodified.

Reply:
weird. lets see how removing sass goes and then i can poke around if you want. If the site is going to be static though you could remove the lazy stuff entirely since it won't help too much.

Reply:
It seems like snap [supports](https://github.com/stereobooster/react-snap/blob/master/doc/recipes.md#split-in-chunks) webpack chunks. Is this what Lazy uses? Based on what I know snap is only semi-static and still depends on being hydrated, generating the most basic html needed for pre-rendering. Wonder if chunks help in this regard. Also glancing over the generated artist htmls I see a lot of `xxxxx.chunk.js`